### DISCUSSION TITLE
DECISION: Remove server reload issue from Milestone "v4"

### DISCUSSION THREAD

**Discussion by plocket** [SuffolkLITLab/ALKiln#533] [General]
## DECISION: Remove #392, server reload issue, from Milestone "v4"

[Comment and discuss in here, give feedback on Slack, see link to feedback process explanation in Slack]

### Why

The main breaking change was converting to using the server API. Now that we have that, we can handle server reload at some point in the future when we have a better idea of how to deal with it.

### What This Looks Like

Remove milestone "v4" from #392 

### Cons

We then don't have an issue connected to milestone "v4" about switching to using the API.

### History
03/30/2022: All 4's and 5's

**Comment by BryceStevenWilley** [comment]
"Will help implement it" just means removing it from the github milestone? If so, then I'm a 4.
